examples/lighting-app/lighting-common/lighting-app.zap

```diff
@@ -0,0 +1,683 @@
+{
+  "writeTime": "Thu Oct 29 2020 15:13:57 GMT-0400 (GMT-04:00)",
+  "featureLevel": 1,
+  "creator": "zap",
+  "keyValuePairs": [
+    {
+      "key": "commandDiscovery",
+      "value": "1"
+    },
+    {
+      "key": "defaultResponsePolicy",
+      "value": "always"
+    },
+    {
+      "key": "filePath",
+      "value": "/Users/jepenven/workspace/connectedhomeip/examples/lock-app/lock-common/lock_app.zap"
+    },
+    {
+      "key": "manufacturerCodes",
+      "value": "0x1002"
+    }
+  ],
+  "package": [
+    {
+      "path": "/Users/jepenven/workspace/zap/zcl-builtin/silabs/zcl.json",
```

Comment:
these paths should be compiler arguments or workspace-relative.  this needs to be addressed in ZAP?